test/integration/singlecluster/controller/jobs/pod/pod_controller_test.go

```diff
@@ -355,6 +356,7 @@ var _ = ginkgo.Describe("Pod controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 
 			ginkgo.When("Pod owner is managed by Kueue", func() {
 				ginkgo.It("Should skip the pod", func() {
+					features.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), features.MultiKueueBatchJobWithManagedBy, false)
```

Comment:
I am going to work on fixing this as a separate PR in #7346.

The integration test was failing with a panic in the job webhook once this feature gate was set to true.

Reply:
please rebase after the merge

Reply:
done.